### ISSUE TITLE
Input not visible while mobile keypad is open

### ISSUE THREAD

**Issue by bjarnef** [Simonwep/pickr#130]
#### Do you want to request a **feature** or report a **bug**? 
Bug

#### What is the current behavior?
When the virtual keypad is open on mobile it is difficult to see what is being entered into the input field.

#### If the current behavior is a bug, please provide the steps to reproduce and create a [JSFiddle](https://jsfiddle.net/Simonwep/qx2Lod6r/).

Can be reproduced in the demo.

#### What is the expected behavior?
To see the overlay plus the input field, while the virtual keypad on mobile is open.
In some of the previous versions it was fixed, but it seems to be an issue in the latest versions.

#### Your environment:
```
Version (see Pickr.version):
Used bundle (es5 or normal one): Normal
Used theme (default is classic): Classic
Browser-version: Chrome 75.0.3770.143
Operating-system: Android 9
```

![Screenshot_20190727-235338_Chrome](https://user-images.githubusercontent.com/2919859/61999868-0b8ab480-b0cb-11e9-84ad-e4a1a6e8bc11.jpg)

![Screenshot_20190727-235358_Chrome](https://user-images.githubusercontent.com/2919859/61999869-10e7ff00-b0cb-11e9-8dc1-9609397bac28.jpg)

**Comment by Simonwep** [comment]
I don't know which issue it was, but back then if pickr could not moved somewhere it should stick on the top of the screen (which happens mostly on mobile devices), but on desktop it looks (at least in scroll-able containers) strange if the popup sticks to one edge :/ I'd need to detect whenever the user is on a mobile device to say it's fine if we stick pickr on the top of the screen...

**Comment by Simonwep** [comment]
OK fixed in b44648f4a3e1fa0f389d286ce54422a454da2232 :)

**Comment by Simonwep** [comment]
It seems like it works fine with media-queries. I'm not sure which would be the best, but at least for now you can easily enter the color on (_most??_) mobile devices. See https://github.com/Simonwep/pickr/commit/fee54521088dde47667b7426a5d36665f144f385#diff-8ed239244ebb411ee69de932344a573bR72
